src/DETHRACE/common/graphics.c

```diff
@@ -1030,8 +1030,8 @@ void DRPixelmapRectangleOnscreenCopy(br_pixelmap* pDest, br_int_16 pDest_x, br_i
 
     source_row_wrap = pSource->row_bytes - pWidth;
     dest_row_wrap = pDest->row_bytes - pWidth;
-    dest_ptr = pDest->pixels + (pDest->row_bytes * pDest_y + pDest_x);
-    source_ptr = pSource->pixels + (pSource->row_bytes * pSource_y + pSource_x);
+    dest_ptr = (char*)pDest->pixels + (pDest->row_bytes * pDest_y + pDest_x);
```

Comment:
most of these are `tU8*` instead of char, which will avoid `-Wpointer-sign` warnings

Reply:
Thanks, I'll address that.

Reply:
Casts should match the pointers they are getting assigned to now.